src/sectors.jl

```diff
@@ -459,3 +459,106 @@ function Base.iterate(s::SectorSet{I}, args...) where {I<:Sector}
     val, state = next
     return convert(I, s.f(val)), state
 end
+
+# Time reversed sector
+struct TimeReversed{I<:Sector} <: Sector
+    a::I
+    function TimeReversed{I}(a::I) where {I<:Sector}
+        if BraidingStyle(I) isa NoBraiding
+            throw(ArgumentError("TimeReversed is not defined for sectors $I with no braiding"))
+        end
+        return new{I}(a)
+    end
+end
+
+sector_rank(::Type{TimeReversed{I}}) where {I<:Sector} = sector_rank(I)
+
+FusionStyle(::Type{TimeReversed{I}}) where {I<:Sector} = FusionStyle(I)
+BraidingStyle(::Type{TimeReversed{I}}) where {I<:Sector} = BraidingStyle(I)
+function Nsymbol(a::TimeReversed{I}, b::TimeReversed{I},
+                 c::TimeReversed{I}) where {I<:Sector}
+    return Nsymbol(a.a, b.a, c.a)
+end
+
+function Fsymbol(a::TimeReversed{I}, b::TimeReversed{I}, c::TimeReversed{I},
+                 d::TimeReversed{I}, e::TimeReversed{I},
+                 f::TimeReversed{I}) where {I<:Sector}
+    return Fsymbol(a.a, b.a, c.a, d.a, e.a, f.a)
+end
+function Rsymbol(a::TimeReversed{I}, b::TimeReversed{I},
+                 c::TimeReversed{I}) where {I<:Sector}
+    # if det(Rsymbol(a.a, b.a, c.a)) ≈ 0.0
+    #     throw(ArgumentError("R-symbol is not invertible for sectors $a, $b, $c"))
+    # end 
+    return Nsymbol(a.a, b.a, c.a) * inv(Rsymbol(a.a, b.a, c.a))
```

Comment:
Am I correct in assuming that the Nsymbol factor is here to fix the issues that arise when the Rsymbol is 0? Given that we are always assuming unitary fusion categories, I think it could be fair to simply replace this with `adjoint(Rsymbol(...))`. I think that also has less issues with the zeros.